### PR TITLE
Update validators.es.xlf

### PR DESCRIPTION
**Pull request by davidpv**
| Q             | A
| ------------- | ---
| Branch?       | master for features / 3.4 up to 4.2 for bug fixes 
| Bug fix?      | yes/no
| New feature?  | yes/no 
| BC breaks?    | no     
| Deprecations? | yes/no 
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.es.xlf [hunk 1]
```diff
@@ -325,6 +325,10 @@
             <trans-unit id="84">
                 <source>This value should be a multiple of {{ compared_value }}.</source>
                 <target>Este valor debería ser múltiplo de {{ compared_value }}.</target>
+            </trans-unit>            
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Este Código de Identificación Bancaria (BIC) no está asociado con el IBAN {{iban}}</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.es.xlf]: Please, maintain the white spaces around the `iban` placeholder name: `{{iban}}` -> `{{ iban }}`  Thanks!

Comment (davidpv) [reply src/Symfony/Component/Validator/Resources/translations/validators.es.xlf]: You're right @javiereguiluz , it's done.

**Review by javiereguiluz — APPROVED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.es.xlf [hunk 2]
```diff
@@ -325,6 +325,10 @@
             <trans-unit id="84">
                 <source>This value should be a multiple of {{ compared_value }}.</source>
                 <target>Este valor debería ser múltiplo de {{ compared_value }}.</target>
+            </trans-unit>            
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Este Código de Identificación Bancaria (BIC) no está asociado con el IBAN {{ iban }}</target>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.es.xlf]: Ending dot is missing.

Comment (davidpv) [reply src/Symfony/Component/Validator/Resources/translations/validators.es.xlf]: done

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @davidpv.

**Pull request merged**